easybuild/easyconfigs/n/nghttp2/nghttp2-1.62.0-GCC-12.3.0.eb

```diff
@@ -0,0 +1,61 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'nghttp2'
+version = '1.62.0'
+
+homepage = 'https://github.com/nghttp2/nghttp2'
+description = """
+This is an implementation of the Hypertext Transfer Protocol version 2 in C.
+
+The framing layer of HTTP/2 is implemented as a reusable C library.
+On top of that, we have implemented an HTTP/2 client, server and proxy.
+We have also developed load test and benchmarking tools for HTTP/2.
+
+An HPACK encoder and decoder are available as a public API."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': SOURCE_TAR_XZ,
+    'git_config': {
+        'url': 'https://github.com/nghttp2',
+        'repo_name': name,
+        'tag': 'v%(version)s',
+        'recursive': True,
+        'keep_git_dir': False,
+    },
+}]
```

Comment:
I think this should just be
```
github_account = 'nghttp2'
source_urls = [GITHUB_RELEASE]
sources = [SOURCE_TAR_XZ]
```

Reply:
Checksum will of course differ, but the content should be the correct and same one as the recursively made one.
That is the actual released source after all.

Reply:
Some file which I found in the previous way of downloading it where actually missing. However, I have changed that for both suggestions.